src_rebuild/GAME/FRONTEND/FEMAIN.C

```diff
@@ -2690,7 +2723,24 @@ int CarSelectScreen(int bSetup)
 	{
 		currSelIndex = pCurrButton->d - 1;
 	}
-
+#ifndef PSX
+#define LAST_START_POINT NUM_START_POINTS-1
+	else if (NumPlayers == 1)
+	{
+		if (fePad & 0x4)
+		{
+			FESound(2);
+			if (--wantedStartPos < -1)
+				wantedStartPos = LAST_START_POINT;
+		}
+		else if (fePad & 0x8)
```

Comment:
maybe just use left/right because L/R shoulder buttons are quite unavailable to keyboard scheme?

Reply:
What do you mean? Is it not left shift/right shift?